src/main/scala/scala/Xgettext.scala

```diff
@@ -30,7 +30,6 @@ class Xgettext(val global: Global) extends Plugin with ScalaVersionAdapter {
   val description = "This Scala compiler plugin extracts and creates gettext.pot file"
   val components  = List[PluginComponent](MapComponent, ReduceComponent)
 
-  val OUTPUT_FILE     = "i18n.pot"
   val HEADER          = """msgid ""
```

Comment:
Thank you for the review!

Ok - Does it really make the compiler much slower - any build system would try to compile only if the source changed, right? However, I could add a force option, too - would that work?

-Abhijit